ts/session/apis/snode_api/swarmPolling.ts

```diff
@@ -617,4 +676,71 @@ export class SwarmPolling {
     // return the cached value
     return this.lastHashes[nodeEdKey][pubkey][namespace];
   }
+
+  public async pollOnceForOurDisplayName(abortSignal?: AbortSignal): Promise<string> {
+    try {
+      if (abortSignal?.aborted) {
+        throw new NotFoundError('[pollOnceForOurDisplayName] aborted right away');
+      }
+
+      const pubkey = UserUtils.getOurPubKeyFromCache();
+      const toPollFrom = await this.getNodesToPollFrom(pubkey.key);
+
+      if (abortSignal?.aborted) {
+        throw new NotFoundError(
+          '[pollOnceForOurDisplayName] aborted after selecting nodes to poll from'
+        );
+      }
+
+      const resultsFromUserProfile = await SnodeAPIRetrieve.retrieveNextMessages(
+        toPollFrom,
+        [''],
+        pubkey.key,
+        [SnodeNamespaces.UserProfile],
+        pubkey.key,
+        null
+      );
```

Comment:
we only want to poll from the namespace of the user profile right?
this.getNodesToPollFrom will return all the user's namespaces, including the 0 one.